### DISCUSSION TITLE
How to update local branch only, on-demand?

### DISCUSSION THREAD

**Discussion by xivarri** [renovatebot/renovate#16238] [Request Help]
Hi - I'm trying to figure out how to use just a small subset of renovate's functionality.
Specifically, I have a Bazel-based project, and just want a command to run on-demand to update my WORKSPACE and MODULE.bazel files in my local repo - either making a new branch or just in the branch I have checked out. I don't need it to push (or even commit), run on a schedule, or anything else.

But because renovate is by default a lot more powerful, I'm struggling with this - e.g. I can't seem to work around providing a "platform", but I only want a local change.

Any help would be appreciated, thanks!

**Comment by rarkins** [comment]
Local mode is not a supported feature

**Comment by xivarri** [comment]
Gotcha, presumably though this is done as a sub-program somewhere within the renovate pipeline, is there any way to extract and run just that part (even if hacky and not guaranteed to not break)?

Like I found the files that somehow handle bazel rules https://github.com/renovatebot/renovate/blob/8901e724f30532ebe7f5da992eb70cc66e8f2df4/lib/modules/manager/bazel/extract.ts
But I have no clue if I can make use of them somehow.

**Reply by viceice** [reply]
not supported

**Comment by mzampetakis** [comment]
Is this something that is still not supported?
I am using this: 

```
docker run --rm \
  -v "$(pwd):/usr/src/app" \
  -v "$HOME/.renovaterc.json:/config" \
  -e LOG_LEVEL=debug \
  renovate/renovate bash -c "git config --global --add safe.directory /usr/src/app && renovate --platform=local --onboarding=false --dry-run"
```

which gives me a list of (go) modules and their available updates (if any). Other that that I would like to have branches created with each one containing the required changes (depending on the provided renovatebot configuration).

Is this something that is still not-supported?  Can I get something like this that I am looking for?
